### PR TITLE
feat: support multiple authorisations to be stored and restored

### PR DESCRIPTION
**Pull request by bobvandevijver**
| Q             | A                                                                                                                         |
|---------------|---------------------------------------------------------------------------------------------------------------------------|
| Bug fix?      | maybe                                                                                                                    |
| New feature?  | maybe                                                                    |
| Deprecations? | no                                                   |
| Issues        | -  |

I have an API description which has multiple authorisation options, which is supported fine by the Swagger interfaces. However, the store/restore functionality always overrides the previous stored value, so only the last input is restored on page load. This change solves that.

As far as I know everything I used should work fine on relevant browsers.
  - Object.keys: https://caniuse.com/mdn-javascript_builtins_object_keys
  - forEach: https://caniuse.com/mdn-javascript_builtins_array_foreach

I do not believe that the build failures are related to this PR....

### CODE REVIEW

File: public/init-swagger-ui.js [hunk 1]
```diff
@@ -25,26 +25,52 @@ function loadSwaggerUI(userOptions = {}) {
 
   const storageKey = 'nelmio_api_auth';
 
-  // if we have auth in storage use it
-  if (sessionStorage.getItem(storageKey)) {
-    try {
-      ui.authActions.authorize(JSON.parse(sessionStorage.getItem(storageKey)));
-    } catch (ignored) {
-      // catch any errors here so it does not stop script execution
+  function getAuthorizationsFromStorage(){
```
Comment (bobvandevijver) [inline public/init-swagger-ui.js:28]:
```suggestion
  function getAuthorizationsFromStorage() {
```

File: public/init-swagger-ui.js [hunk 2]
```diff
@@ -25,26 +25,52 @@ function loadSwaggerUI(userOptions = {}) {
...
+    if (sessionStorage.getItem(storageKey)) {
+      try {
+        return JSON.parse(sessionStorage.getItem(storageKey))
```
Comment (bobvandevijver) [inline public/init-swagger-ui.js:31]:
```suggestion
        return JSON.parse(sessionStorage.getItem(storageKey));
```

**Comment by bobvandevijver** [comment]
@DjordyKoert Anything I can do to get this merged?

**Comment by DjordyKoert** [comment]
Sorry for the late response.

It's generally quite cumbersome to test / validate that any changes to the javascript code will function 😢

**Comment by DjordyKoert** [comment]
Thank you for the contribution at least ❤️

**Pull request merged**